dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -0,0 +1,18 @@
+[
+    {
+        "TUNNEL_DECAP_TABLE:IPINIP_TUNNEL" : {
+            "tunnel_type":"IPINIP",
+            "src_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
+            "dst_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
+            "dscp_mode":"uniform",
```

Comment:
pipe currently is not supported on Mellanox platforms. Will left it "uniform" for Mellanox platform and pipe for other